platform/windows/os_windows.cpp

```diff
@@ -805,7 +805,12 @@ void OS_Windows::process_key_events() {
 
 				k->set_pressed(ke.uMsg == WM_KEYDOWN);
 
-				k->set_scancode(KeyMappingWindows::get_keysym(ke.wParam));
+				if ((ke.lParam & (1 << 24)) && (ke.wParam == VK_RETURN)) {
```

Comment:
Some other keys also have bit 24 set, but afaik numpad Enter is only one without unique keycode.
[MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646280(v=vs.85).aspx)